src/GitHub.InlineReviews/PullRequestStatusBarPackage.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Windows;
+using System.Threading;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.ComponentModelHost;
+using GitHub.VisualStudio;
+using GitHub.InlineReviews.Services;
+using Task = System.Threading.Tasks.Task;
+
+namespace GitHub.InlineReviews
+{
+    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [Guid(Guids.PullRequestStatusPackageId)]
+    [ProvideAutoLoad(Guids.GitSccProviderId)]
+    public class PullRequestStatusBarPackage : AsyncPackage
+    {
+        /// <summary>
+        /// Initialize the PR status UI on Visual Studio's status bar.
+        /// </summary>
+        protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
+        {
+            var componentModel = (IComponentModel)await GetServiceAsync(typeof(SComponentModel));
+            var exportProvider = componentModel.DefaultExportProvider;
+            var pullRequestStatusManager = exportProvider.GetExportedValue<IPullRequestStatusBarManager>();
```

Comment:
MEF calls need to happen on the main thread, and there's no guarantee that this method is going to be called on the main thread, especially once the new auto-load changes come (https://docs.microsoft.com/en-us/visualstudio/extensibility/how-to-use-asyncpackage-to-load-vspackages-in-the-background). For things that are required during package loading, they should be registered as services instead to avoid having to hit MEF on startup (see GitHubPackage.cs)

Reply:
I've changed `IVSGitExt` to be a VS services with no MEF dependencies.